helm/hpcc/values.yaml

```diff
@@ -197,9 +197,21 @@ vaults:
 
   esp:
 
+  ## The keys for code signing may be imported from the vault.  Multiple keys may be imported.
+  ## gpg keys may be imported as follows:
+  ## vault kv put secret/codeSign/gpg-private-key-1 passphrase=<passphrase> private=@<private_key_file1>
+  ## vault kv put secret/codeSign/gpg-private-key-2 passphrase=<passphrase> private=@<private_key_file2>
   codeSign:
-
+    - name: codesign-private-keys
+      url: http://${env.VAULT_SERVICE_HOST}:${env.VAULT_SERVICE_PORT}/secret/data/codeSign/${secret}
```

Comment:
is this an example?  Should it be commoned out it the final PR?

Reply:
This URL works for the default vault configuration.  Users should be able to pull and start default the Hashicorp vault image, enable for k8,  upload the keys to the vault and use the service for signing and verification.  (There is a small caveat in that the egress rules will need to be updated).  Is there any harm in keeping this in?

Reply:
What will happen if vault is not installed?  Will the base coe stall waiting for a response, or generate an error if it is not available?

Reply:
@shamser ?

Reply:
If vault isn't installed or configured, an error will be written to the ESP log: Vault auth error - vault: codesign-private-keys - vault access token not provided.  I have commented out the vault configuration. @ghalliday